core/src/main/java/io/questdb/cairo/TableUtils.java

```diff
@@ -1842,6 +1923,65 @@ private static CharSequence getCharSequence(MemoryMR metaMem, long memSize, long
         return metaMem.getStrA(offset);
     }
 
+    private static MatViewDefinition loadMatViewDefinitionMessage(final ReadableBlock mem, final TableToken matViewToken) {
```

Comment:
nit: rename loadMatViewDefinitionMessage -> loadMatViewDefinition

Reply:
Changed in 89e3665b3d9d55d9e442abc04cca05ab12915062